include/PetriEngine/PQL/Contexts.h

```diff
@@ -235,6 +236,9 @@ namespace PetriEngine {
 
             glp_prob* makeBaseLP() const;
 
+            mutable std::vector<uint32_t> xs;
```

Comment:
`xs` seems like a funky naming scheme? Also please use the `_` prefix for fields.